viz/charts/linkerd-viz/README.md

```diff
@@ -127,7 +127,7 @@ Kubernetes: `>=1.21.0-0`
 | metricsAPI.tolerations | string | `nil` | Tolerations section, See the [K8S documentation](https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/) for more information |
 | namespaceMetadata.image.name | string | `"extension-init"` | Docker image name for the namespace-metadata instance |
 | namespaceMetadata.image.pullPolicy | string | defaultImagePullPolicy | Pull policy for the namespace-metadata instance |
-| namespaceMetadata.image.registry | string | `"cr.l5d.io/linkerd"` | Docker registry for the namespace-metadata instance |
+| namespaceMetadata.image.registry | string | defaultRegistry | Docker registry for the namespace-metadata instance |
```

Comment:
FYI I was curious how this inaccuracy got into main and filed a PR to prevent it in the future: https://github.com/linkerd/linkerd2/pull/10781